includes/class-amp-reader-themes.php

```diff
@@ -0,0 +1,546 @@
+<?php
+/**
+ * Fetches and formats data for AMP reader themes.
+ *
+ * @package AMP
+ * @since 1.6.0
+ */
+
+/**
+ * Class AMP_Reader_Themes.
+ *
+ * @since 1.6.0
+ */
+final class AMP_Reader_Themes {
+	/**
+	 * Formatted theme data.
+	 *
+	 * @since 1.6.0
```

Comment:
There's an overabundance of `@since` tags in this file.

I'd recommend adding a `@since` tag to the class docblock only for now, and adding them on adaptations when they diverge from the initially published class.

Especially for private properties in a final class like here, this is currently just noise.